__tests__/index-test.js

```diff
@@ -2472,4 +2497,75 @@ it('Bananas incompatible plugin @react-navigation source snapshot matches', () =
     },
   );
   expect(code).toMatchInlineSnapshot(BananasStandardOutputNoAttributes);
+});
+
+it('Bananas custom attribute names let component override element with setFSTagName', () => {
```

Comment:
Yeah, it is a bit confusing name.. but I think it is actually overriding, in the sense that the component name is overriding the element name. The normal output from `BananasStandardOutputWithAttributes` has [element set to Image and component set to Bananas](https://github.com/fullstorydev/fullstory-babel-plugin-annotate-react/blob/14c02ac28a186c9bcc4e1f7da0f7754c5d481987/__tests__/index-test.js#L1151-L1152).

So, part of what this test is validating is that it chooses the component name rather than the element name for `fsTagName`. 

Perhaps I should add a couple more tests, one on something that only has an element and one that only has a component?

Reply:
I think I understand what you're saying. Maybe just replace the test description with:
`'setFSTagName sets fsTagName on component with its dataComponent value'`?